hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -756,48 +756,78 @@
     jobs:
         - 'kubernetes-e2e-gce-trusty-{suffix}'
 
-# Jobs that run e2e tests on GKE with Trusty as node image on the release-1.1 branch.
+# Jobs that run e2e tests on GKE with Trusty as node image on the release-1.2 branch.
 - project:
     name: kubernetes-e2e-gke-trusty
-    trigger-job: 'kubernetes-build-1.1'
+    trigger-job: 'kubernetes-build-1.2'
     test-owner: 'wonderfly@google.com'
-    branch: 'release-1.1'
-    runner: '{old-runner-1-1}'
-    post-env: ''
+    provider-env: |
+        {gke-provider-env}
     emails: 'wonderfly@google.com,qzheng@google.com'
     suffix:
+        - 'gke-trusty-test':
+            description: 'Run E2E tests on GKE test endpoint.'
+            timeout: 480
+            job-env: |
+                export PROJECT="kubekins-e2e-gke-trusty-test"
+                export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
+                export JENKINS_USE_SERVER_VERSION="y"
+                export E2E_NAME="jkns-gke-e2e-test-trusty"
```

Comment:
Since each of these is running in a separate project, please don't use `E2E_NAME`, and let it default.